mlflow/langchain/__init__.py

```diff
@@ -153,6 +155,7 @@ def save_model(
     persist_dir=None,
     example_no_conversion=False,
     model_config=None,
+    streamable=None,
```

Comment:
Does `None` mean like "auto"?

Reply:
Yes, it's no-op if user doesn't call predict_stream. If they call it, we detect if predict_stream is supported